### ISSUE TITLE
VIsual enum editor adds extra (blank) values

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#12277]
When using the visual enum editor to add less than four enum values, it will add blank values. For instance, `` `e_test` enum('a','b','','') NOT NULL``

**Comment by ibennetch** [comment]
One question about this, though; is there a scenario where users would want to add a blank value (`''`)? Fixing this would break that.

**Comment by nijel** [comment]
Just to add reference, there is PR on this: https://github.com/phpmyadmin/phpmyadmin/pull/12318

**Comment by ragnerok** [comment]
this has been fixed right ?

**Comment by ibennetch** [comment]
@ragnerok No, at this point it has not yet. We got a bit held up trying to decide between intentional blank values or ones generated by the form.

**Comment by ragnerok** [comment]
ok

**Comment by jfactotum** [comment]
@ibennetch, it seems this one should be closed based on #13064, yes? --jsw

**Comment by ibennetch** [comment]
@jfactotum No, at least I don't think so. That one was closed because it's a duplicate of this one, but the issue is still not fixed.

**Comment by kkarol012** [comment]
Hello. Is issue still occurring?

**Comment by ibennetch** [comment]
@kkarol012 This may not be a great first issue (because there's already been one attempt to fix it that hasn't solved all use-cases), but it is open and available. You certainly may work on it.

**Comment by ibennetch** [comment]
I have two ideas for how to fix this.

1. Start the form with an empty field and let the user manually add however many values they want.

2. We could incorporate a checkbox to the side of each field that would say "blank value" or "include blank value" or perhaps "empty string", and the user could click that checkbox to force the inclusion of the empty value, otherwise the form should ignore the empty ones.

**Comment by williamdes** [comment]
> "empty string", and the user could click that checkbox to force the inclusion of the empty value, otherwise the form should ignore the empty ones.

LGTM

**Comment by kkarol012** [comment]
Second option looks good to me too. I'll try to implement that way.
